test/unit/component/parser/json/BpmnJsonParser.flowNode.test.ts

```diff
@@ -207,10 +195,20 @@ describe.each([
 
   if (expectedShapeBpmnElementKind === ShapeBpmnElementKind.GATEWAY_EVENT_BASED) {
     it(`should convert as Shape, when a ${bpmnKind} (with/without instantiate) is an attribute (as array) of 'process'`, () => {
-      const json = {
+      const json: BpmnJsonModel = {
         definitions: {
           targetNamespace: '',
-          process: {},
+          process: {
+            [bpmnKind]: [
+              { id: `${bpmnKind}_id_1` },
+              { id: `${bpmnKind}_id_2`, eventGatewayType: 'Exclusive' },
+              // forbidden by the BPMN spec, only valid when 'instantiate: true'
+              { id: `${bpmnKind}_id_3`, eventGatewayType: 'Parallel' },
```

Comment:
**nitpick:** I suggest to update the comment as it is uncleared to which part the comment applies.
Previously, it was on the same line as the related parameter, which is no more the case.